### PR TITLE
1015-move research/ to project-MONAI/research-contributions

### PR DESCRIPTION
**Pull request by wyli**
part of #1015 https://github.com/Project-MONAI/research-contributions

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: docs/source/highlights.md [hunk 1]
```diff
@@ -218,13 +218,13 @@ We always welcome contributions in forms of comments, suggestions, and code impl
 The generic patterns/modules identified from the research prototypes will be integrated into MONAI core functionality.
 
 ### COPLE-Net for COVID-19 Pneumonia Lesion Segmentation
-[A reimplementation](https://github.com/Project-MONAI/MONAI/tree/master/research/coplenet-pneumonia-lesion-segmentation) of the COPLE-Net originally proposed by:
+[A reimplementation](https://monai.io/research/coplenet-pneumonia-lesion-segmentation) of the COPLE-Net originally proposed by:
```
Comment (Nic-Ma) [inline docs/source/highlights.md:221]: I see a `re-direction` page for 2 secs, from monai.io to github repo.
So I think maybe it's better to use github link directly?
https://github.com/Project-MONAI/research-contributions/tree/master/coplenet-pneumonia-lesion-segmentation

Thanks.

File: docs/source/highlights.md [hunk 2]
```diff
@@ -218,13 +218,13 @@ We always welcome contributions in forms of comments, suggestions, and code impl
...
 
 G. Wang, X. Liu, C. Li, Z. Xu, J. Ruan, H. Zhu, T. Meng, K. Li, N. Huang, S. Zhang. (2020) "A Noise-robust Framework for Automatic Segmentation of COVID-19 Pneumonia Lesions from CT Images." IEEE Transactions on Medical Imaging. 2020. [DOI: 10.1109/TMI.2020.3000314](https://doi.org/10.1109/TMI.2020.3000314)
 ![image](../images/coplenet.png)
 
 ### LAMP: Large Deep Nets with Automated Model Parallelism for Image Segmentation
-[A reimplementation](https://github.com/Project-MONAI/MONAI/tree/master/research/lamp-automated-model-parallelism) of the LAMP system originally proposed by:
+[A reimplementation](https://monai.io/research/lamp-automated-model-parallelism) of the LAMP system originally proposed by:
```
Comment (Nic-Ma) [inline docs/source/highlights.md:227]: Same as above.

**Review by Nic-Ma — APPROVED** [review]
Looks good to me except the links.
Thanks.

File: docs/source/highlights.md (hunk 1 above)
Comment (wyli) [reply docs/source/highlights.md:221]: the idea is to use https://monai.io/research/coplenet-pneumonia-lesion-segmentation as a permanent link, not depending on the specific folder structure of `Project-MONAI/research-contributions`

**Pull request merged**